Formula/tldr.rb

```diff
@@ -0,0 +1,20 @@
+class Tldr < Formula
+  desc "Simplified and community-driven man pages"
+  homepage "https://tldr-pages.github.io"
+  url "https://github.com/tldr-pages/tldr-cpp-client/archive/v1.2.0.tar.gz"
+  sha256 "db2e70b3f4209c67b5b9896e052bf3d9e77570dcb04d0c9932776aa44cca9d3e"
+  head "https://github.com/tldr-pages/tldr-cpp-client.git"
+
+  depends_on "curl"
+  depends_on "libzip"
+  depends_on "pkg-config" => :build
+
+  def install
+    system "make"
```

Comment:
Alternatively, or probably even better, would be 

    system "make", "DEBUG=-O2"

This will enable some, safe, optimizations, and disable the debugging flags.